### PR TITLE
r/glue_catalog_table - add arn attribute

### PR DESCRIPTION
**Pull request by DrFaust92**
add disappears test

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #13624, #13527

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
resource_aws_glue_catalog_table - add arn attribute
```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSGlueCatalogTable_'
--- PASS: TestAccAWSGlueCatalogTable_recreates (64.76s)
--- PASS: TestAccAWSGlueCatalogTable_basic (63.52s)
--- PASS: TestAccAWSGlueCatalogTable_full (62.75s)
--- PASS: TestAccAWSGlueCatalogTable_update_addValues (108.52s)
--- PASS: TestAccAWSGlueCatalogTable_update_replaceValues (110.66s)
--- PASS: TestAccAWSGlueCatalogTable_StorageDescriptor_EmptyConfigurationBlock (42.91s)
--- PASS: TestAccAWSGlueCatalogTable_StorageDescriptor_SerDeInfo_EmptyConfigurationBlock (48.60s)
--- PASS: TestAccAWSGlueCatalogTable_StorageDescriptor_SkewedInfo_EmptyConfigurationBlock (56.27s)
--- PASS: TestAccAWSGlueCatalogTable_disappears (60.08s)
```

### CODE REVIEW

File: aws/resource_aws_glue_catalog_table.go [hunk 1]
```diff
@@ -271,6 +276,15 @@ func resourceAwsGlueCatalogTableRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error reading Glue Catalog Table: %s", err)
 	}
 
+	tableArn := arn.ARN{
+		Partition: meta.(*AWSClient).partition,
+		Service:   "glue",
+		Region:    meta.(*AWSClient).region,
+		AccountID: meta.(*AWSClient).accountid,
```
Comment (bflad) [inline aws/resource_aws_glue_catalog_table.go:283]: Can you please add this resource to the `skip_requesting_account_id` documentation in `website/docs/index.html.markdown`? Thanks!

File: website/docs/r/glue_catalog_table.html.markdown [hunk 2]
```diff
@@ -135,6 +135,14 @@ The following arguments are supported:
 * `skewed_column_value_location_maps` - (Optional) A list of values that appear so frequently as to be considered skewed.
 * `skewed_column_values` - (Optional) A map of skewed values to the columns that contain them.
 
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Catalog ID, Database name and of the name table.
+* `arn` - The ARN of the Glue Table.
+
```
Comment (bflad) [inline website/docs/r/glue_catalog_table.html.markdown:144]: Minor fixes 😄 

```suggestion
* `id` - Catalog ID, database name, and table name.
* `arn` - Amazon Resource Name (ARN) of the Glue Table.
```

File: aws/resource_aws_glue_catalog_table.go (hunk 1 above)
Comment (DrFaust92) [reply aws/resource_aws_glue_catalog_table.go:283]: done

File: aws/resource_aws_glue_catalog_table_test.go [hunk 3]
```diff
@@ -31,15 +31,15 @@ func TestAccAWSGlueCatalogTable_recreates(t *testing.T) {
 				PreConfig: func() {
```
Comment (bflad) [inline aws/resource_aws_glue_catalog_table_test.go:31]: Created followup issue for this older disappears testing style: https://github.com/terraform-providers/terraform-provider-aws/issues/13820

**Pull request merged**